.github/workflows/lint.yml

```diff
@@ -12,4 +12,5 @@ jobs:
         linters: "[\"clj-kondo\" \"kibit\" \"eastwood\"]"
         github_token: ${{ secrets.GITHUB_TOKEN }}
         runner: ":leiningen"
+        base_sha: ${{ github.event.before }}
```

Comment:
I'm sorry if I misunderstood, but I guess `github.event.pull_request.base.sha` is more suitable for PR events since files eventually changed in a PR is more important than files changed in the last commit of a PR.